File: java/program/Pet.java

public class Pet extends Animal {
    private String owner;
    private boolean vaccinated;
    
    public Pet() {
    }
    
    public Pet(String species, String name, String owner, boolean vaccinated) {
        super(species, name);
        this.owner = owner;
        this.vaccinated = vaccinated;
    }
    
    public void setOwner(String owner) {
        this.owner = owner;
    }
    
    public void setVaccinated(boolean vaccinated) {
        this.vaccinated = vaccinated;
    }
    
    public String getOwner() {
        return this.owner;
    }
    
    public boolean isVaccinated() {
        return this.vaccinated;
    }
}

File: python/program/Pet.py

from Animal import Animal

class Pet(Animal):
    def __init__(self, species="", name="", owner="", vaccinated=False):
        super().__init__(species, name)
        self.__owner = owner
        self.__vaccinated = vaccinated
    
    def set_owner(self, owner):
        self.__owner = owner
    
    def set_vaccinated(self, vaccinated):
        self.__vaccinated = vaccinated
    
    def get_owner(self):
        return self.__owner
    
    def is_vaccinated(self):
        return self.__vaccinated